ibis/backends/pandas/trace.py

```diff
@@ -117,13 +117,21 @@ def _log_trace(func, start=None):
 
 
 def trace(func):
-    """ Return a function decorator that wraped the decorated function with
+    """Return a function decorator that wraped the decorated function with
     tracing.
     """
     _trace_funcs.add(func.__name__)
 
     @functools.wraps(func)
     def traced_func(*args, **kwargs):
+        # Unfortunately, this function can be called before the `ibis.pandas`
+        # attribute has ever been accessed, which means the trace configuration
+        # option might never get registered and will raise an error. Accessing
+        # the pandas attribute here forces the option initialization
```

Comment:
@datapythonista :point_up: the bug :)

Reply:
Another thing to note is that this bug doesn't admit an obvious unit test, AFAICT. I was only able to trigger it when parallelizing tests with a medium-ish number of processes (32). Something like `pytest-randomly` might help.